openhands/runtime/impl/action_execution/action_execution_client.py

```diff
@@ -48,7 +48,7 @@
 
 def _is_retryable_error(exception):
     return isinstance(
-        exception, (httpx.RemoteProtocolError, httpcore.RemoteProtocolError)
+        exception, (httpx.ReadError, httpx.RemoteProtocolError)
```

Comment:
why did we remove `httpcore.RemoteProtocolError`

Reply:
Because `httpx` should wrap `httpcore.RemoteProtocolError` as `httpx.RemoteProtocolError`, making the explicit httpcore check redundant.